.size-limit.js

```diff
@@ -13,7 +13,7 @@ module.exports = [
     name: 'All Integrations - CDN',
     path: 'dist/legacy/js-integrations/*.min.js',
     gzip: true,
-    limit: '390.34 kB',
+    limit: '400.0 kB',
```

Comment:
Did we increase this manually ? Is SDK team ok with this change ?

Reply:
No it was taken according to the extra size required. As far as sdk team is concerned I am yet to take approval from them.